test/test_checks.py

```diff
@@ -1,12 +1,10 @@
+from flake8_polyfill.version import version_info
 from flake8_quotes import QuoteChecker
 import os
 import subprocess
 from unittest import expectedFailure, TestCase
 
 
-FLAKE8_VERSION = os.environ.get('FLAKE8_VERSION', '2')
```

Comment:
I've changed this because when I locally use Flake8 3.x this variable might not be set. As we require `flake8_polyfill` anyway I figured it's okay to use it to get a replacement.